### DISCUSSION TITLE
Feature request : Could you create a deployment script for a DeepStack lxc container ?

### DISCUSSION THREAD

**Discussion by Mrdindon** [tteck/Proxmox#412] [Ideas]
I'm running many of your containers and really enjoy how easy it is to deploy.
I'm also running DeepStack AI server currently as a docker directly on the proxmox host (not ideal I know)
https://www.deepstack.cc

I was wondering if you had any idea or it would be feasible to script a lxc container dedicated to that ?

**Comment by timdonovanuk** [comment]
@Mrdindon There are steps to run it from scratch here: https://github.com/johnolafenwa/DeepStack under "Running and Testing Locally Without Building". I'd say try it on a Debian LXC first.

**Comment by Mrdindon** [comment]
@timdonovanuk I tried these steps already without success on debian 11 lxc :(

**Comment by timdonovanuk** [comment]
Ah :(

What about trying the commands that Docker runs? Taking a quick look it seems like docker runs the following:

https://github.com/johnolafenwa/DeepStack/blob/dev/base/Dockerfile.cpu
then 
https://github.com/johnolafenwa/DeepStack/blob/dev/Dockerfile.cpu

I don't have a great knowledge of Dockerfiles (and this seems to be quite a complex example) but I've been successful creating my own LXC files for a few other bits of software by basically running everything within Dockerfile

**Reply by Mrdindon** [reply]
Same here. I'm not good with dockers. And I really like the scripted method that is used here to deploy various solutions. I think it would fit well. :)

**Reply by Mrdindon** [reply]
I also tried these steps without success :(
